src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,110 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
```

Comment:
It's good practice to avoid using concrete implementation classes in declarations if possible. Consider using `List` interface instead of `LinkedList` for the `buckets` array declaration.